api/src/test/java/org/eclipse/microprofile/rest/client/BuilderImpl2.java

```diff
@@ -32,6 +33,10 @@ public RestClientBuilder baseUri(URI uri) {
         throw new IllegalStateException("not implemented");
     }
 
+    public RestClientBuilder executorService(ExecutorService executor) {
```

Comment:
can be in `AbstractBuilder`

Reply:
I can move it - but it seemed like the `AbstractBuilder` class mainly had methods from `Configurable` while the sub-classes had the `RestClientBuilder` -specific methods.